<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerData</class>
 <widget class="QMainWindow" name="LayerData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1354</width>
    <height>952</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="30,20">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>InfluxDB</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,18,10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="3,1,5,1,3">
         <item>
          <spacer name="horizontalSpacer_15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>数据预览</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_16">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>数据显示</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_17">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="20,1,20">
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_20">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QChartView" name="graphicsInflux"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17" stretch="10,1,10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Influx交互命令：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_4"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="3,1,3,1,3,1,3">
             <item>
              <widget class="QPushButton" name="pushButton_8">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>连接数据库</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_12">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_9">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>添加</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_13">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_10">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>删除</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_14">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_13">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>修改</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_18">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,1,1">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>筛选类型：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>筛选值：</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_11">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,1,1">
             <item>
              <widget class="QComboBox" name="comboBox_2"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_5"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_14">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>筛选</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>通道信息设置</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="18,1,10">
       <item>
        <widget class="QTableWidget" name="ChannelWidget"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_19">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>42</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1">
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,3,3,1">
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>定时采集</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>采集时间：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_3"/>
            </item>
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>s</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1,1">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>采样点数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_2"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_15">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>导入数据</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="3,3,1,1,4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>分析频率：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Hz</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>28</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_16">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>导出数据</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="home"/>
   <addaction name="data"/>
   <addaction name="compute"/>
   <addaction name="config"/>
   <addaction name="connect"/>
   <addaction name="clean"/>
   <addaction name="more"/>
  </widget>
  <action name="clean">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/33.png</normaloff>:/logo/33.png</iconset>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
   <property name="toolTip">
    <string>清除</string>
   </property>
  </action>
  <action name="home">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/24.png</normaloff>:/logo/24.png</iconset>
   </property>
   <property name="text">
    <string>目录</string>
   </property>
   <property name="toolTip">
    <string>目录</string>
   </property>
  </action>
  <action name="connect">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/8.png</normaloff>:/logo/8.png</iconset>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
   <property name="toolTip">
    <string>连接</string>
   </property>
  </action>
  <action name="setting">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/3.png</normaloff>:/logo/3.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
  </action>
  <action name="monitor">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/13.png</normaloff>:/logo/13.png</iconset>
   </property>
   <property name="text">
    <string>监测</string>
   </property>
   <property name="toolTip">
    <string>监测</string>
   </property>
  </action>
  <action name="data">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/15.png</normaloff>:/logo/15.png</iconset>
   </property>
   <property name="text">
    <string>数据</string>
   </property>
   <property name="toolTip">
    <string>数据</string>
   </property>
  </action>
  <action name="more">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/28.png</normaloff>:/logo/28.png</iconset>
   </property>
   <property name="text">
    <string>更多</string>
   </property>
   <property name="toolTip">
    <string>更多</string>
   </property>
  </action>
  <action name="model">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/32.png</normaloff>:/logo/32.png</iconset>
   </property>
   <property name="text">
    <string>模型</string>
   </property>
   <property name="toolTip">
    <string>模型</string>
   </property>
  </action>
  <action name="analysis">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/9.png</normaloff>:/logo/9.png</iconset>
   </property>
   <property name="text">
    <string>分析</string>
   </property>
   <property name="toolTip">
    <string>分析</string>
   </property>
  </action>
  <action name="compute">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/16.png</normaloff>:/logo/16.png</iconset>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
   <property name="toolTip">
    <string>计算</string>
   </property>
  </action>
  <action name="config">
   <property name="icon">
    <iconset resource="../res/main.qrc">
     <normaloff>:/logo/6.png</normaloff>:/logo/6.png</iconset>
   </property>
   <property name="text">
    <string>配置</string>
   </property>
   <property name="toolTip">
    <string>配置</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/main.qrc"/>
 </resources>
 <connections/>
</ui>
